aggsender/flows/feature_maxl2blocknumber.go

```diff
@@ -0,0 +1,122 @@
+package flows
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/agglayer/aggkit/aggsender/types"
+)
+
+var (
+	ErrMaxL2BlockNumberExceededInARetryCert = errors.New("max L2 block number exceeded in a retry certificate")
+	ErrComplete                             = errors.New("all certs send, no more certificates can be sent")
+	ErrBuildParamsIsNil                     = errors.New("buildParams is nil")
+)
+
+type FeatureMaxL2BlockNumber struct {
```

Comment:
To me this makes more sense than naming a feature: FeatureN

```suggestion
type MaxL2BlockNumberLimiter struct {
```

Reply:
I prefer the name of `Feature` because it means that is a optional behaviour

Reply:
I agree with Goran, prefixing `Feature` is kind of weird, we don't have any other "feature" prefixed like that

Reply:
The reason to call this a feature is because all the logic are inside this object. Also you can instantiate or not (it acts as a featureFlag). But, anyway, if the name is confusing, no problem I'll rename it

Reply:
Renamed: 890fa2b48ed875bd0379674470a53baa17863e9d